Makefile

```diff
@@ -47,6 +47,11 @@ build: vendor generate
 	mkdir -p target
 	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags "$(GO_LDFLAGS)" -o target/pg-bifrost github.com/Nextdoor/pg-bifrost.git/main
 
+build_mac: vendor generate
+	@echo "Creating GO binary"
+	mkdir -p target
+	go build -o target/pg-bifrost github.com/Nextdoor/pg-bifrost.git/main
```

Comment:
Add `CGO_ENABLED=0 GOOS=darwin GOARCH=amd64` and ldflags so the two are consistent

Reply:
👍